packages/components/src/pages/ChatSearch.vue

```diff
@@ -655,6 +789,27 @@ $border-color: darken($gray4, 10%);
     }
   }
 
+  .search-wrapper {
+    display: grid;
+    grid-template-rows: min-content min-content 1fr;
+
+    max-height: 100vh;
+    background-color: darken(#292c39, 10%);
+
+    .pinned-items-wrapper,
+    .context-wrapper {
+      display: flex;
+      flex-direction: column;
+      overflow-y: auto;
+      margin-bottom: 0;
+    }
+
+    .pinned-items-wrapper .search-container {
+      max-height: 33vh;
+      border-bottom: 1px dashed $white;
+    }
```

Comment:
This border does not lay on the bottom margins of the 'pinned items' container, so it makes the 'selected by Navie' header appear have a lot of a lot of margin on top.
![image](https://github.com/user-attachments/assets/c55e4117-4a28-4f2f-a621-de62a4855cd3)

![image](https://github.com/user-attachments/assets/400e3bf8-c235-444d-b379-059a4fe43b2d)